src/kOS.Safe/Exceptions/KOSLookupFailException.cs

```diff
@@ -30,7 +30,7 @@ public class KOSLookupFailException: KOSException
         /// <param name="exist">True if the reason for the message is that the thing exists at
         ///   other times, but is just not available at the moment.</param>
         public KOSLookupFailException(string category, string lookupName, object obj, bool exist = false) :
-            base(String.Format( (exist ? TERSE_NOT_EXIST_MSG_FMT : TERSE_NOT_EXIST_MSG_FMT), category, lookupName.ToUpper(), obj.ToString()) )
+            base(String.Format( (exist ? TERSE_DOES_EXIST_MSG_FMT : TERSE_NOT_EXIST_MSG_FMT), category, lookupName.ToUpper(), obj.ToString()) )
```

Comment:
Not related to the issue, but I noticed it when debugging the issue - this exception message has been wrong for ages (always gave the "not exist" message even when the field does exist but is invisible.)